### PR TITLE
Fix image break in iOS 14 where images won't show

### PR DESCRIPTION
**Pull request by HeyImChris**
#### Please select one of the following
- [ ] I am removing an existing difference between facebook/react-native and microsoft/react-native-macos :thumbsup:
- [x] I am cherry-picking a change from Facebook's react-native into microsoft/react-native-macos :thumbsup:
- [ ] I am making a fix / change for the macOS implementation of react-native
- [x] I am making a change required for Microsoft usage of react-native

## Summary

When images are built with Xcode 12 Beta for iOS 14, images don't display in some scenarios. This was fixed in RN 0.63 but we should bring the fix in now so our downstream clients on 0.62 have function images.

[Discussion](https://github.com/facebook/react-native/issues/29279)

This doesn't affect RN 0.60 so we only need to fix it in master and not the stable branch.

## Changelog

[iOS] [Bug] - Fix iOS 14 image render bug

## Test Plan

Images appear with the fix and don't appear without the fix (see attachments). I built for iOS 14 with Xcode 11 and Xcode 12 Beta 5 environments.

before

after

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-macos/pull/573)

### CODE REVIEW

**Comment by acoates-ms** [comment]
Actually, you'll need to apply this fix to the `0.62-stable` branch.  `master` is no longer tracking stable releases, which allows merges to continue without destabilizing stable releases.

**Review by harrieshin — APPROVED** [review]

**Comment by HeyImChris** [comment]
> Actually, you'll need to apply this fix to the `0.62-stable` branch. `master` is no longer tracking stable releases, which allows merges to continue without destabilizing stable releases.

Will merge into both 0.62 stable and master. The fix went in at 0.63.2 so 0.63.0-stable wouldn't have it

**Review by acoates-ms — APPROVED** [review]

**Review by anandrajeswaran — APPROVED** [review]

File: RNTester/Podfile.lock [hunk 1]
```diff
@@ -348,34 +348,34 @@ EXTERNAL SOURCES:
...
+  React-RCTSettings: 5c4b741b59a4547f8f002a3c17bc9b817298c8e0
+  React-RCTTest: 5afe9d3c7ab66dfd2ca5145f9189c0a85f2fb34b
+  React-RCTText: 09b6d9d7363064d23380f50fc01af3ab75131cf7
+  React-RCTVibration: 8135beb48e3065fbcbe6417b9a6bbbc884c02a78
+  ReactCommon: a377fdc9faecc5c9408bcd07ce997703c0ce22ff
+  Yoga: 7d784f976888661a2cea9333ab06125b1133e435
 
 PODFILE CHECKSUM: 1a5700c13b2bc8ffc5ed4282d795e8909c883f63
 
-COCOAPODS: 1.9.3
```
Comment (tom-un) [inline RNTester/Podfile.lock:381]: nit: need to update your cocoapods `brew update cococapods`

**Review by tom-un — APPROVED** [review]

File: RNTester/Podfile.lock (hunk 1 above)
Comment (alloy) [reply RNTester/Podfile.lock:381]: Alternatively, RNTester comes with a `Gemfile` nowadays, meaning that if you do the following you should always be using the same CP version that others are:

```bash
gem install bundler
bundle install
bundle exec pod install
```

**Pull request merged**